<launch>
    <node pkg="netft_rdt_driver" name="ft11" type="netft_node" args="192.168.0.127" respawn="true">
        <remap from='netft_data' to='/force_torque_ft11' />
    </node>

    <node name="static_tf_ft11_broadcaster" pkg="tf"
        type="static_transform_publisher"
        args="0 0 0 0 0 0 1 world force_torque_ft11 50"
        respawn="false" output="screen"/>
</launch>

